kieker-tools/build.gradle

```diff
@@ -21,6 +21,10 @@ dependencies {
 	compile 'org.apache.commons:commons-math3:3.6.1'	// used by OPAD
 	compile 'org.apache.commons:commons-lang3:3.6'		// used by OPAD
 	compile configurations.resultJar
+	
+	compile group: 'log4j', name: 'log4j', version: '1.2.17'
```

Comment:
Do we really want to include a logging implementation? I agree that for the *tools* projects this may make sense. However, to stay aligned with our [Readme](https://github.com/kieker-monitoring/kieker#debugging-and-logging-kiekers-execution) we should consider to either switch here to Logback or update the Readme accordingly. To my knowledge, Logback can somehow be seen as Log4J's successor but i'm not sure about this and do not know anything about the advantages and disadvantages of both frameworks.